File: sujet_machine/exemples/a_table/skeleton/a_table.scala
object a_table
{
  
  def a_table0(n : Int, m : Int, salle : Array[Array[Char]]){
    /* Inserez votre code ici */
  }
  
  def main(args : Array[String])
  {
    var (n, m) = readf2("{0,number} {1,number}").asInstanceOf[(Long, Long)] match { case x => (x._1.toInt, x._2.toInt) };
    var dg :Array[Array[Char]] = new Array[Array[Char]](n);
    for (dh <- 0 to n - 1)
        dg(dh) = readLine().toCharArray();
    var salle: Array[Array[Char]] = dg;
    a_table0(n, m, salle);
  }
  
}

File: sujet_machine/exemples/a_table/skeleton/a_table.java
import java.util.*;

public class a_table
{
  static Scanner scanner = new Scanner(System.in);
  static int[] read_int_line()
  {
    String[] s = scanner.nextLine().split(" ");
    int[] out = new int[s.length];
    for (int i = 0; i < s.length; i++)
      out[i] = Integer.parseInt(s[i]);
    return out;
  }

  static void a_table0(int n, int m, char[][] salle)
  {
    //  Inserez votre code ici 
  }
  public static void main(String args[])
  {
    int[] by = read_int_line();
    int n = by[0];
    int m = by[1];
    char[][] bz = new char[n][];
    for (int ca = 0; ca < n; ca++)
        bz[ca] = scanner.nextLine().toCharArray();
    char[][] salle = bz;
    a_table0(n, m, salle);
  }
  
}

